build/build/src/ide/web/fonts.rs

```diff
@@ -72,6 +72,65 @@ pub async fn install_enso_font_for_html(
     Ok(())
 }
 
+pub async fn install_enso_font_for_html_2(
+    cache: &Cache,
+    octocrab: &Octocrab,
+    css_basepath: &str,
+    output_path: impl AsRef<Path>,
+    css_output_path: impl AsRef<Path>,
+) -> Result {
+    let output_path = output_path.as_ref();
+    ide_ci::fs::tokio::create_dir_if_missing(output_path).await?;
```

Comment:
use 
```
let html_fonts = vec![…];
``` 
for initializing vectors.

Though it is not a best choice for lookups. (not really a problem for ~10 elements)